curriculum/challenges/english/blocks/workshop-colorful-boxes/68f23761964a40289166e2fb.md

```diff
@@ -0,0 +1,116 @@
+---
+id: 68f23761964a40289166e2fb
+title: Step 9
+challengeType: 0
+dashedName: step-9
+---
+
+# --description--
+
+Inside each `.box` element, add an `h3` element.
+
+Inside the six `h3` elements add the following texts, in order:
+
+Box 1
+Box 2
+Box 3
+Box 4
+Box 5
+Box 6
```

Comment:
These must be `h2`s because there's no reason to jump from `h1` to `h3`. All the tests and following steps should be changed accordingly.
```suggestion
Inside each `.box` element, add an `h2` element with the text `Box` followed by a space and a sequential number starting from `1`.
```